src/preprocessors/preprocessor.ts

```diff
@@ -22,7 +32,13 @@ export function preprocessor(code: string, options: PrettierOptions): string {
         parserOptions.allowReturnOutsideFunction = true;
     }
 
-    const ast = babelParser(code, parserOptions);
+    let ast;
+    try {
+        ast = babelParser(code, parserOptions);
```

Comment:
I said in the original issue that it was babel.traverse that was throwing, but I mispoke; it was actually the parse that happens first.  So, I only wrapped the parse itself.